src/Loco.php

```diff
@@ -197,7 +197,13 @@ public function import(MessageCatalogueInterface $catalogue, array $options = []
         $locale = $catalogue->getLocale();
         foreach ($this->projects as $project) {
             foreach ($project->getDomains() as $domain) {
-                $data = XliffConverter::catalogueToContent($catalogue, $domain);
+                $options = [];
```

Comment:
We now have a `$options` parameters for this method, you can safely remove this line.

Reply:
True. And only set the `$options['default_locale'] = 'en';` if `!isset($options['default_locale'])`.

Reply:
Good point:
Should we override the `default_locale` if it's an existing option when using `id` as index?
I'd say yes cause I don't have any use case in mind were you need to use a specific locale as source target for this kind of translation...
WDYT?

Reply:
I don't think this is a decision I can make. As @odolbeau says it doesn't feel like it makes sense to set the `default_locale` to a specific value when using `id` as the index, but at the same time allowing this might make it easier for people to support very specific use cases.

This feels like a design decision, choosing between allowing specific use cases versus protecting developers from misconfiguring the adapter.

Reply:
@Nyholm WDYT?